src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php

```diff
@@ -94,6 +94,7 @@ public function __construct()
             new ResolveHotPathPass(),
             new ResolveNoPreloadPass(),
             new AliasDeprecatedPublicServicesPass(),
+            new RemoveBuildParametersPass(),
```

Comment:
As this removes things, shouldn't it be a removing pass instead ?

Reply:
I get the rationale, but on the other hand, this would prevent after-removing passes from using build-time parameter. We'd better allow them I believe.

Reply:
Would probably deserve to be written down as a code comment to prevent it from being moved by mistake.

Reply:
if that's the intent, it should probably be registered as a low priority instead of the default priority of 0 (otherwise, most after-removing pass not defined in the DI component will still run after it)

Reply:
Done.